Gems/AWSGameLift/Code/AWSGameLiftServer/Source/AWSGameLiftServerManager.cpp

```diff
@@ -404,46 +458,26 @@ namespace AWSGameLift
         }
     }
 
-    AZ::IO::Path AWSGameLiftServerManager::GetExternalSessionCertificate()
+    AZ::IO::Path AWSGameLiftServerManager::GetInternalSessionCertificate()
     {
-        auto certificateOutcome = m_gameLiftServerSDKWrapper->GetInstanceCertificate();
-        if (certificateOutcome.IsSuccess())
-        {
-            return AZ::IO::Path(certificateOutcome.GetResult().GetCertificatePath().c_str());
-        }
-        else
-        {
-            AZ_Error(AWSGameLiftServerManagerName, false, AWSGameLiftServerInstanceCertificateErrorMessage);
-            return AZ::IO::Path();
-        }
+        // GameLift doesn't support it, return empty path
+        return AZ::IO::Path();
     }
 
-    AZ::IO::Path AWSGameLiftServerManager::GetExternalSessionPrivateKey()
-    {
-        auto certificateOutcome = m_gameLiftServerSDKWrapper->GetInstanceCertificate();
+    AZ::IO::Path AWSGameLiftServerManager::GetExternalSessionCertificate()
+    {   
+        auto certificateOutcome = m_gameLiftServerSDKWrapper->GetComputeCertificate();
         if (certificateOutcome.IsSuccess())
         {
-            return AZ::IO::Path(certificateOutcome.GetResult().GetPrivateKeyPath().c_str());
+            return AZ::IO::Path(certificateOutcome.GetResult().GetCertificatePath().c_str());
         }
         else
         {
-            AZ_Error(AWSGameLiftServerManagerName, false, AWSGameLiftServerInstancePrivateKeyErrorMessage);
+            AZ_Error(AWSGameLiftServerManagerName, false, AWSGameLiftServerInstanceCertificateErrorMessage);

```

Comment:
Where is `AWSGameLiftServerInstanceCertificateErrorMessage` coming from? Does the `certificateOutcome` contain an error message that should be used instead?

Reply:
It is located in `AWSGameLiftServerManager.h` on line 59
```
        static constexpr const char AWSGameLiftServerInstanceCertificateErrorMessage[] =
            "Failed to locate Amazon GameLift TLS certificate file.";
```

I added the error message as well.

```
            AZ_Error(
                AWSGameLiftServerManagerName,
                false,
                AZStd::string::format(
                    "%s - %s",
                    AWSGameLiftServerInstanceCertificateErrorMessage, 
                    certificateOutcome.GetError().GetErrorMessage().c_str()
                ).c_str()
            );
```